docs/src/developers_guide/release.rst

```diff
@@ -11,6 +11,20 @@ The summary below is of the main areas that constitute the release.  The final
 section details the :ref:`iris_development_releases_steps` to take.
 
 
+.. _release-manager:
+
+Release Manager
+---------------
+A Release Manager will be nominated for each release of Iris. This role involves:
+
+* deciding which features and bug fixes should be included in the release
+* managing the project board for the release
+* updating the iris dev community via the discussion board
+
+The Release Manager will make the release, ensuring that all the steps outlined
+on this page are completed.
+
```

Comment:
I would suggest also linking thru to the discussion categories https://www-avd/~itkt/iris-docs-dev/src/_build/html/developers_guide/release.html stating this is where the release manager will document intent and capture any discussion points about a release.

Perhaps also update the Iris 3.2 and 3.3 discussions first post to link to this release guidance for reference.